src/tests/ftest/control/dmg_telemetry_io_basic.yaml

```diff
@@ -0,0 +1,43 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-A
+timeout: 150
+server_config:
+  name: daos_server
+  servers:
+    targets: 2
+pool:
+  name: daos_server
+  scm_size: 1G
+  control_method: dmg
+container:
+  type: POSIX
+  control_method: daos
+block_sizes: [10M, 500M]
+transfer_sizes: [1M]
+telemetry_metrics:
+  io_test_metrics: [
```

Comment:
This brings up two questions:
1) Is there a reason we're not looking at all the `engine_io_*` metrics - even if it is to just verify that a value is returned for each name?
2) Should we really be redefining (even part of) the `TelemetryUtils.ENGINE_IO_METRICS` list here?  No there are two places that need to be maintained if the list/names are updated.

I think we should keep any list definitions inside the `TelemetryUtils` class to maintain consistency in use.  If it makes sense to define the `engine_io_*` metrics into smaller lists, then lets do that in the class.

Reply:
Thanks for the suggestion.
Have updated the script to query all the io metrics and verify with the specified io test metrics.

Reply:
I think we are going to have similar problem when we have more telemetry testcases, where maintaining consistency will be a challenge. Maybe consider breaking down the ENGINE_IO_METRICS list in telemetry_utils to their logical areas? If you need the entire engine io metrics, you can always extend the list. But having a smaller logical definition will allow you to maintain it in one place and also possibly read and verify what you need. get_io_metrics right now goes through the entire ENGINE_IO_METRICS list.
ENGINE_IO_DTX_METRICS
ENGINE_IO_LATENCY_METRICS
:
ENGINE_IO_OPS_UPDATE_METRICS